docs/calendar.md

```diff
@@ -334,7 +336,8 @@ date.toString(); // => '2020-06-28[u-ca=islamic]'
 - `options` (object): An object with properties representing options for the operation.
   The following options are recognized:
   - `largestUnit` (optional string): The largest unit of time to allow in the resulting `Temporal.Duration` object.
-    Valid values are `'auto'`, `'years'`, `'months'`, and `'days'`.
+    Valid values are `'auto'`, `'year'`, `'month'`, and `'day'`.
+    Plural values (e.g. `'days'`) are accepted though not recommended.
```

Comment:
I think it's better to continue doing what we were doing before — just not mention the unrecommended alternative.

(Same goes for other places below where this is added)

Reply:
I was following the precedent used in [Intl.RelativeTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/formatToParts) docs on MDN: 

> Unit to use in the relative time internationalized message. Possible values are: "year", "quarter", "month", "week", "day", "hour", "minute", "second". Plural forms are also permitted.

Still think I should remove the note here?  I don't have a strong opinion either way-- happy to go with your preference.

Reply:
I don't feel strongly about this, it's just a general preference for brevity when all else is equal.

Reply:
OK I'll remove. It can always be added later if there's objections.